### ISSUE TITLE
How to use a variable in the array?

### ISSUE THREAD

**Issue by vit-goncharov** [taverntesting/tavern#42]
I'm trying to use my variable value  in the array but it doesn't work. E.g.:
```
json:
        settings: []
        testlist: ["{test_id:d}"]
```

**Comment by vit-goncharov** [comment]
Send
```
json:
        settings: []
        testlist:
          - "{test_id:d}"
```

Print on server
['{test_id:d}']

**Comment by benhowes** [comment]
Closed by #43
